src/index.js

```diff
@@ -0,0 +1,57 @@
+import format from './formatter';
+
+import glob from 'fast-glob';
+
+import { lintFile } from './lint';
+
+import { readFile } from 'fs';
+import { promisify } from 'util';
+const readFileAsync = promisify(readFile);
+
+const rules = [
+  tag => {
+    if (tag.tagName !== 'include') return;
+    // The include path is the part before the comma and/or whitespace, the params are the rest
+    const includePath = tag.tagValue.split(/[\s,]+/)[0];
+    if (includePath.startsWith("'") && includePath.endsWith("'")) return;
+    return {
+      ruleId: 'paths-should-be-quoted',
+      message: 'Expected path to be quoted',
+      source: tag.source,
+      severity: 'error'
+    };
+  },
+  tag => {
+    // The path argument is the part before the comma and/or whitespace, the params are the rest
+    const params = tag.tagValue.split(/[\s,]+/);
+    // Params [1] is the first param after the path argument
+    if (params[1] === 'with') {
+      return {
+        ruleId: 'no-with',
+        message: 'Arguments should be comma-separated instead of using `with`',
+        source: tag.source,
+        severity: 'error'
+      };
+    }
+  }
+];
+
+const main = async () => {
+  const args = process.argv.slice(2);
+  const userGlobs = args.filter(arg => !arg.startsWith('-'));
+  const files = await glob(userGlobs.length > 0 ? userGlobs : ['**/*.liquid'], {
+    ignore: ['node_modules/**/*']
```

Comment:
I do not know if this matters but I accidentally ran `fluidlint` at a higher level directory than I meant to run it in. It ended up checking a file that exists within the `node_modules` directory. Is that the expected result?

```
$ fluidlint

  projects/baptist-health/repos/baptist-health-patterns/node_modules/@11ty/eleventy/test/stubs/includer.liquid
  ✖  Expected path to be quoted                                 paths-should-be-quoted
     {% include included %}
```

Reply:
Good catch 👍